main.go

```diff
@@ -883,6 +884,13 @@ func (task *TaskRun) run() error {
 	if err != nil {
 		return WorkerShutdown(err)
 	}
+	// using NewWriterLevel(...) instead of NewWriter(...) to be explicit about
+	// compression level, so easy to change later
+	gzipLogWriter, err := gzip.NewWriterLevel(logFileHandle, gzip.DefaultCompression)
+	if err != nil {
+		return WorkerShutdown(err)
+	}
+	gzipLogWriter.Name = "live_backing.log"
```

Comment:
Really we are not setting anything special....the log is just the task output piped to a gzip module for compression, and then piped to a temp file [1]. Then we're just setting three headers when uploading [2].

[1] https://github.com/taskcluster/docker-worker/blob/master/lib/features/bulk_log.js#L33
[2] https://github.com/taskcluster/docker-worker/blob/master/lib/features/bulk_log.js#L53-L55